### PR TITLE
Redirect IPv6 inbound traffic to specified port

### PR DESCRIPTION
**Pull request by ChangyuWang**
Please provide a description for what this PR is for.

Redirect ipv6 inbound traffic to target ISTIO_IN_REDIRECT

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ x ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: tools/packaging/common/istio-iptables.sh [hunk 1]
```diff
@@ -544,6 +544,8 @@ if [ -n "${ENABLE_INBOUND_IPV6}" ]; then
             ip6tables -t ${table} -A ISTIO_INBOUND -p tcp --dport "${port}" -j RETURN
         done
         fi
+        # Redirect other inbound traffic
+        ip6tables -t ${table} -A ISTIO_INBOUND -p tcp -j ISTIO_IN_REDIRECT
```
Comment (hzxuzhonghu) [inline tools/packaging/common/istio-iptables.sh]: This is done in L398

Comment (ChangyuWang) [reply tools/packaging/common/istio-iptables.sh]: It is for ipv6, not ipv4.

Comment (hzxuzhonghu) [reply tools/packaging/common/istio-iptables.sh]: ic

**Comment by ChangyuWang** [comment]
@howardjohn OK, I will

**Comment by ChangyuWang** [comment]
@howardjohn done

**Comment by howardjohn** [comment]
Can someone familiar with this code review? @sbezverk @rlenglet @lambdai

**Comment by sbezverk** [comment]
@howardjohn looks good to me

**Comment by rlenglet** [comment]
@howardjohn do you still have concerns blocking this PR?

**Comment by rlenglet** [comment]
Once this is merged, I will integrate it into istio/cni as well.

**Pull request merged**